feat(ci): implement new single-script e2e test workflow

seershan opened the pull request:
Thank you for submitting a pull request to the WrongSecrets Party!

What kind of changes does this PR include?

- [x] Fixes or refactors
- [x] A new feature

Checklist:

- [x] All the contributions made are solely the work of me and my co-authors
- [x] I tested the changes in this PR (if applicable)
- [x] I added tests to ensure my change works (if applicable)
- [x] The PR passes pre-commit hooks and automated tests

---

This pull request adds a suite of Cypress End-to-End tests and a new GitHub Action to run them, fulfilling the requirements of issue #284. It also includes the necessary fixes to the deployment scripts and configuration to get the local and CI environments working.

This is a clean replacement for the previous pull request (#970).

Closes #284

Copilot committed:
Initial plan

Copilot committed:
feat: upgrade to Node.js 24 and update Jest to v30 with deprecation fixes

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>

Copilot committed:
feat: update dependencies and fix selectUnit function logic

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>

Copilot committed:
feat: complete Node.js 24 upgrade and dependency modernization

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>

Copilot committed:
Initial assessment of failing tests and precommit issues

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>

Copilot committed:
Fix Jest 30 test isolation issue and resolve lockfile synchronization

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>

Copilot committed:
Upgrade TypeScript to latest version (5.9.2)

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>

Copilot committed:
Fix GitHub Actions and Docker build with --legacy-peer-deps for TypeScript 5.9.2 compatibility

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>

Copilot committed:
Fix GitHub Actions compatibility: add Node.js 24 setup to workflows and enforce engines requirements

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>

Copilot committed:
Fix precommit trailing whitespace issue in AdminPage.js

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>

seershan committed:
docs: add detailed windows setup guide

renovate[bot] committed:
chore(config): migrate config renovate.json

seershan committed:
feat(ci): implement new single-script e2e test workflow

seershan committed:
Merge branch 'main' into fix/ci-e2e-script

seershan committed:
fix(ci): reduce cpu and memory request for minikube in ci

seershan committed:
fix(ci): reduce cpu and memory request for minikube in ci

seershan committed:
ci: run deploy script in background and wait for pods

seershan committed:
ci: run deploy script in background and wait for pods

dependabot[bot] committed:
build(deps): bump axios in /wrongsecrets-balancer/ui

Bumps [axios](https://github.com/axios/axios) from 1.11.0 to 1.12.0.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.11.0...v1.12.0)

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.12.0
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

renovate[bot] committed:
chore(deps): update dependency jest to v30.1.3

renovate[bot] committed:
chore(deps): update pre-commit hook renovatebot/pre-commit-hooks to v41.114.0

renovate[bot] committed:
chore(deps): update terraform azurerm to v4.44.0

renovate[bot] committed:
chore(deps): update dependency @eslint/js to v9.35.0

renovate[bot] committed:
chore(deps): update dependency globals to v16.4.0

renovate[bot] committed:
chore(deps): update terraform-aws

dependabot[bot] committed:
build(deps): bump tar-fs from 3.1.0 to 3.1.1 in /cleaner

Bumps [tar-fs](https://github.com/mafintosh/tar-fs) from 3.1.0 to 3.1.1.
- [Commits](https://github.com/mafintosh/tar-fs/compare/v3.1.0...v3.1.1)

---
updated-dependencies:
- dependency-name: tar-fs
  dependency-version: 3.1.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump tar-fs from 3.1.0 to 3.1.1 in /wrongsecrets-balancer

Bumps [tar-fs](https://github.com/mafintosh/tar-fs) from 3.1.0 to 3.1.1.
- [Commits](https://github.com/mafintosh/tar-fs/compare/v3.1.0...v3.1.1)

---
updated-dependencies:
- dependency-name: tar-fs
  dependency-version: 3.1.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

renovate[bot] committed:
chore(deps): update dependency @eslint/js to v9.36.0

renovate[bot] committed:
chore(deps): update dependency @google-cloud/storage to v7.17.1

renovate[bot] committed:
chore(deps): update dependency eslint to v9.37.0

renovate[bot] committed:
chore(deps): update dependency google-auth-library to v10.4.0

commjoen commented:
Hi @seershan , it's been a while! Hope you are doing great! 
Can you please have a look at the feedback and the failing checks? Feel free to DM me over slack if you are having trouble resolving the issues.

seershan commented:
Hello @commjoen actually im out of station currently, I will start again by Tuesday.
Sorry for the delay.

commjoen commented:
all good! Have a good time and will see the commits later 👍

seershan committed:
Update guides/WINDOWS_SETUP.md

Co-authored-by: Jeroen Willemsen <jeroenwillemsen2001@gmail.com>

Copilot committed:
Initial plan

Copilot committed:
Fix preview deployment workflow variable expansion

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>

Copilot committed:
Fix GITHUB_OUTPUT variable expansion to prevent empty comment body

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>

Copilot committed:
Skip pushing images for PRs from forks to prevent permission errors

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>

commjoen committed:
Merge branch 'main' into fix/ci-e2e-script

Copilot committed:
Initial plan

Copilot committed:
Add .npmrc to fix TypeScript peer dependency conflict

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>

Copilot committed:
Fix npm dependency conflict in wrongsecrets-balancer/ui by adding .npmrc

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>

commjoen committed:
Merge branch 'main' into fix/ci-e2e-script

renovate[bot] committed:
chore(deps): update dependency openid-client to v6.8.1

commjoen committed:
fix npm ci

Copilot committed:
Initial plan

Copilot committed:
Upgrade Express to v5.1.0 and fix compatibility issues

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>

commjoen committed:
Merge branch 'main' into fix/ci-e2e-script

renovate[bot] committed:
chore(deps): update dependency @google-cloud/iam to v2.3.1

renovate[bot] committed:
chore(deps): update dependency typescript to v5.9.3

renovate[bot] committed:
chore(deps): update dependency @kubernetes/client-node to v1.4.0

renovate[bot] committed:
chore(deps): update dependency @google-cloud/secret-manager to v6.1.1

renovate[bot] committed:
chore(deps): update dependency @google-cloud/storage to v7.17.2

renovate[bot] committed:
chore(deps): update dependency axios to v1.12.2

renovate[bot] committed:
chore(deps): update dependency jest to v30.2.0

renovate[bot] committed:
chore(deps): update dependency kubernetes-sigs/aws-alb-ingress-controller to v2.14.0

renovate[bot] committed:
chore(deps): update dependency python to 3.14

renovate[bot] committed:
chore(deps): update dependency react-router-dom to v7.9.4

renovate[bot] committed:
chore(deps): update dependency winston to v3.18.3

renovate[bot] committed:
chore(deps): update dependency google-auth-library to v10.4.1

renovate[bot] committed:
chore(deps): update pre-commit

renovate[bot] committed:
chore(deps): update terraform azurerm to v4.48.0

renovate[bot] committed:
chore(deps): update dependency kubernetes-sigs/aws-alb-ingress-controller to v2.14.1

renovate[bot] committed:
chore(deps): update dependency react-router-dom to v7.9.5

renovate[bot] committed:
chore(deps): update dependency @eslint/js to v9.39.0

renovate[bot] committed:
chore(deps): update dependency axios to v1.13.1

renovate[bot] committed:
chore(deps): update dependency eslint to v9.39.0

renovate[bot] committed:
chore(deps): update dependency globals to v16.5.0

renovate[bot] committed:
chore(deps): update dependency google-auth-library to v10.5.0

renovate[bot] committed:
chore(deps): update dependency minimatch to v10.1.1

renovate[bot] committed:
chore(deps): update dependency kubernetes-sigs/aws-alb-ingress-controller to v2.15.0

renovate[bot] committed:
chore(deps): update pre-commit

renovate[bot] committed:
chore(deps): update dependency bcryptjs to v3.0.3

renovate[bot] committed:
chore(deps): update dependency @google-cloud/storage to v7.17.3

renovate[bot] committed:
chore(deps): update dependency axios to v1.13.2

renovate[bot] committed:
chore(deps): update dependency eslint to v9.39.1

renovate[bot] committed:
chore(deps): update dependency react-router-dom to v7.9.6

dependabot[bot] committed:
build(deps): bump body-parser in /wrongsecrets-balancer

Bumps [body-parser](https://github.com/expressjs/body-parser) from 2.2.0 to 2.2.1.
- [Release notes](https://github.com/expressjs/body-parser/releases)
- [Changelog](https://github.com/expressjs/body-parser/blob/master/HISTORY.md)
- [Commits](https://github.com/expressjs/body-parser/compare/v2.2.0...v2.2.1)

---
updated-dependencies:
- dependency-name: body-parser
  dependency-version: 2.2.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump node-forge in /wrongsecrets-balancer/ui

Bumps [node-forge](https://github.com/digitalbazaar/forge) from 1.3.1 to 1.3.2.
- [Changelog](https://github.com/digitalbazaar/forge/blob/main/CHANGELOG.md)
- [Commits](https://github.com/digitalbazaar/forge/compare/v1.3.1...v1.3.2)

---
updated-dependencies:
- dependency-name: node-forge
  dependency-version: 1.3.2
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump glob from 10.4.5 to 10.5.0 in /wrongsecrets-balancer

Bumps [glob](https://github.com/isaacs/node-glob) from 10.4.5 to 10.5.0.
- [Changelog](https://github.com/isaacs/node-glob/blob/main/changelog.md)
- [Commits](https://github.com/isaacs/node-glob/compare/v10.4.5...v10.5.0)

---
updated-dependencies:
- dependency-name: glob
  dependency-version: 10.5.0
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

renovate[bot] committed:
chore(deps): update dependency prettier to v3.7.3

renovate[bot] committed:
chore(deps): update pre-commit hook antonbabenko/pre-commit-terraform to v1.104.0

renovate[bot] committed:
chore(deps): update dependency express to v5.2.0 [security]

renovate[bot] committed:
chore(deps): update terraform azurerm to v4.54.0

dependabot[bot] committed:
build(deps): bump express from 5.1.0 to 5.2.0 in /wrongsecrets-balancer

Bumps [express](https://github.com/expressjs/express) from 5.1.0 to 5.2.0.
- [Release notes](https://github.com/expressjs/express/releases)
- [Changelog](https://github.com/expressjs/express/blob/master/History.md)
- [Commits](https://github.com/expressjs/express/compare/v5.1.0...v5.2.0)

---
updated-dependencies:
- dependency-name: express
  dependency-version: 5.2.0
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

renovate[bot] committed:
chore(deps): update terraform-aws

renovate[bot] committed:
chore(deps): update dependency kubernetes-sigs/aws-alb-ingress-controller to v2.16.0

dependabot[bot] committed:
build(deps): bump jws from 4.0.0 to 4.0.1 in /wrongsecrets-balancer

Bumps [jws](https://github.com/brianloveswords/node-jws) from 4.0.0 to 4.0.1.
- [Release notes](https://github.com/brianloveswords/node-jws/releases)
- [Changelog](https://github.com/auth0/node-jws/blob/master/CHANGELOG.md)
- [Commits](https://github.com/brianloveswords/node-jws/compare/v4.0.0...v4.0.1)

---
updated-dependencies:
- dependency-name: jws
  dependency-version: 4.0.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

renovate[bot] committed:
chore(deps): update dependency @eslint/eslintrc to v3.3.3

renovate[bot] committed:
chore(deps): update dependency cookie to v1.1.1

renovate[bot] committed:
chore(deps): update dependency @google-cloud/storage to v7.18.0

renovate[bot] committed:
chore(deps): update dependency react-router-dom to v7.10.1

dependabot[bot] committed:
build(deps): bump glob from 10.4.5 to 10.5.0 in /cleaner

Bumps [glob](https://github.com/isaacs/node-glob) from 10.4.5 to 10.5.0.
- [Changelog](https://github.com/isaacs/node-glob/blob/main/changelog.md)
- [Commits](https://github.com/isaacs/node-glob/compare/v10.4.5...v10.5.0)

---
updated-dependencies:
- dependency-name: glob
  dependency-version: 10.5.0
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump js-yaml from 4.1.0 to 4.1.1

Bumps [js-yaml](https://github.com/nodeca/js-yaml) from 4.1.0 to 4.1.1.
- [Changelog](https://github.com/nodeca/js-yaml/blob/master/CHANGELOG.md)
- [Commits](https://github.com/nodeca/js-yaml/compare/4.1.0...4.1.1)

---
updated-dependencies:
- dependency-name: js-yaml
  dependency-version: 4.1.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

renovate[bot] committed:
chore(deps): update terraform azurerm to v4.56.0

renovate[bot] committed:
chore(deps): update terraform-gcp to ~> 6.50.0

renovate[bot] committed:
chore(deps): update dependency eslint to v9.39.2

renovate[bot] committed:
chore(deps): update dependency express to v5.2.1

renovate[bot] committed:
chore(deps): update dependency prettier to v3.7.4

renovate[bot] committed:
chore(deps): update dependency winston to v3.19.0

renovate[bot] committed:
chore(deps): update terraform aws to v6.27.0

dependabot[bot] committed:
build(deps-dev): bump express in /wrongsecrets-balancer/ui

Bumps [express](https://github.com/expressjs/express) from 4.21.2 to 4.22.1.
- [Release notes](https://github.com/expressjs/express/releases)
- [Changelog](https://github.com/expressjs/express/blob/v4.22.1/History.md)
- [Commits](https://github.com/expressjs/express/compare/4.21.2...v4.22.1)

---
updated-dependencies:
- dependency-name: express
  dependency-version: 4.22.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump js-yaml in /wrongsecrets-balancer/ui

Bumps  and [js-yaml](https://github.com/nodeca/js-yaml). These dependencies needed to be updated together.

Updates `js-yaml` from 3.14.1 to 3.14.2
- [Changelog](https://github.com/nodeca/js-yaml/blob/master/CHANGELOG.md)
- [Commits](https://github.com/nodeca/js-yaml/compare/3.14.1...3.14.2)

Updates `js-yaml` from 4.1.0 to 4.1.1
- [Changelog](https://github.com/nodeca/js-yaml/blob/master/CHANGELOG.md)
- [Commits](https://github.com/nodeca/js-yaml/compare/3.14.1...3.14.2)

---
updated-dependencies:
- dependency-name: js-yaml
  dependency-version: 3.14.2
  dependency-type: indirect
- dependency-name: js-yaml
  dependency-version: 4.1.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

renovate[bot] committed:
chore(deps): update terraform azurerm to v4.58.0

renovate[bot] committed:
chore(deps): update actions/cache action to v5

renovate[bot] committed:
chore(deps): update actions/setup-python action to v6

dependabot[bot] committed:
build(deps): bump lodash in /wrongsecrets-balancer/ui

Bumps [lodash](https://github.com/lodash/lodash) from 4.17.21 to 4.17.23.
- [Release notes](https://github.com/lodash/lodash/releases)
- [Commits](https://github.com/lodash/lodash/compare/4.17.21...4.17.23)

---
updated-dependencies:
- dependency-name: lodash
  dependency-version: 4.17.23
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump qs from 6.14.0 to 6.14.1 in /cleaner

Bumps [qs](https://github.com/ljharb/qs) from 6.14.0 to 6.14.1.
- [Changelog](https://github.com/ljharb/qs/blob/main/CHANGELOG.md)
- [Commits](https://github.com/ljharb/qs/compare/v6.14.0...v6.14.1)

---
updated-dependencies:
- dependency-name: qs
  dependency-version: 6.14.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump react-router and react-router-dom

Bumps [react-router](https://github.com/remix-run/react-router/tree/HEAD/packages/react-router) to 7.13.0 and updates ancestor dependency [react-router-dom](https://github.com/remix-run/react-router/tree/HEAD/packages/react-router-dom). These dependencies need to be updated together.


Updates `react-router` from 7.10.1 to 7.13.0
- [Release notes](https://github.com/remix-run/react-router/releases)
- [Changelog](https://github.com/remix-run/react-router/blob/main/packages/react-router/CHANGELOG.md)
- [Commits](https://github.com/remix-run/react-router/commits/react-router@7.13.0/packages/react-router)

Updates `react-router-dom` from 7.10.1 to 7.13.0
- [Release notes](https://github.com/remix-run/react-router/releases)
- [Changelog](https://github.com/remix-run/react-router/blob/main/packages/react-router-dom/CHANGELOG.md)
- [Commits](https://github.com/remix-run/react-router/commits/react-router-dom@7.13.0/packages/react-router-dom)

---
updated-dependencies:
- dependency-name: react-router
  dependency-version: 7.13.0
  dependency-type: indirect
- dependency-name: react-router-dom
  dependency-version: 7.13.0
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

renovate[bot] committed:
chore(deps): update actions/setup-node action to v6

renovate[bot] committed:
chore(deps): update actions/github-script action to v8

renovate[bot] committed:
chore(deps): update actions/checkout action to v6

renovate[bot] committed:
chore(deps): update dependency kubernetes-sigs/aws-alb-ingress-controller to v2.17.1

dependabot[bot] committed:
build(deps): bump lodash in /wrongsecrets-balancer

Bumps [lodash](https://github.com/lodash/lodash) from 4.17.21 to 4.17.23.
- [Release notes](https://github.com/lodash/lodash/releases)
- [Commits](https://github.com/lodash/lodash/compare/4.17.21...4.17.23)

---
updated-dependencies:
- dependency-name: lodash
  dependency-version: 4.17.23
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump qs from 6.14.0 to 6.14.1 in /wrongsecrets-balancer

Bumps [qs](https://github.com/ljharb/qs) from 6.14.0 to 6.14.1.
- [Changelog](https://github.com/ljharb/qs/blob/main/CHANGELOG.md)
- [Commits](https://github.com/ljharb/qs/compare/v6.14.0...v6.14.1)

---
updated-dependencies:
- dependency-name: qs
  dependency-version: 6.14.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

renovate[bot] committed:
chore(deps): update dependency prettier to v3.8.1

renovate[bot] committed:
chore(deps): update dependency styled-components to v6.3.8

renovate[bot] committed:
chore(deps): update dependency supertest to v7.2.2

renovate[bot] committed:
chore(deps): update terraform random to ~> 3.8.0

renovate[bot] committed:
chore(deps): update terraform random to v3.8.1

dependabot[bot] committed:
build(deps-dev): bump webpack in /wrongsecrets-balancer/ui

Bumps [webpack](https://github.com/webpack/webpack) from 5.101.2 to 5.105.0.
- [Release notes](https://github.com/webpack/webpack/releases)
- [Changelog](https://github.com/webpack/webpack/blob/main/CHANGELOG.md)
- [Commits](https://github.com/webpack/webpack/compare/v5.101.2...v5.105.0)

---
updated-dependencies:
- dependency-name: webpack
  dependency-version: 5.105.0
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump jsonpath in /wrongsecrets-balancer/ui

Bumps [jsonpath](https://github.com/dchester/jsonpath) from 1.1.1 to 1.2.1.
- [Commits](https://github.com/dchester/jsonpath/commits/1.2.1)

---
updated-dependencies:
- dependency-name: jsonpath
  dependency-version: 1.2.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump fast-xml-parser and @google-cloud/storage

Bumps [fast-xml-parser](https://github.com/NaturalIntelligence/fast-xml-parser) to 5.3.6 and updates ancestor dependency [@google-cloud/storage](https://github.com/googleapis/nodejs-storage). These dependencies need to be updated together.


Updates `fast-xml-parser` from 4.5.3 to 5.3.6
- [Release notes](https://github.com/NaturalIntelligence/fast-xml-parser/releases)
- [Changelog](https://github.com/NaturalIntelligence/fast-xml-parser/blob/master/CHANGELOG.md)
- [Commits](https://github.com/NaturalIntelligence/fast-xml-parser/commits/v5.3.6)

Updates `@google-cloud/storage` from 7.18.0 to 7.19.0
- [Release notes](https://github.com/googleapis/nodejs-storage/releases)
- [Changelog](https://github.com/googleapis/nodejs-storage/blob/main/CHANGELOG.md)
- [Commits](https://github.com/googleapis/nodejs-storage/compare/v7.18.0...v7.19.0)

---
updated-dependencies:
- dependency-name: fast-xml-parser
  dependency-version: 5.3.6
  dependency-type: indirect
- dependency-name: "@google-cloud/storage"
  dependency-version: 7.19.0
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

renovate[bot] committed:
chore(deps): update dependency minimatch to v10.2.2

renovate[bot] committed:
chore(deps): update dependency styled-components to v6.3.11

renovate[bot] committed:
chore(deps): update terraform azurerm to v4.61.0

renovate[bot] committed:
chore(deps): update eslint

dependabot[bot] committed:
build(deps-dev): bump js-yaml in /wrongsecrets-balancer

Bumps [js-yaml](https://github.com/nodeca/js-yaml) from 3.14.1 to 3.14.2.
- [Changelog](https://github.com/nodeca/js-yaml/blob/master/CHANGELOG.md)
- [Commits](https://github.com/nodeca/js-yaml/compare/3.14.1...3.14.2)

---
updated-dependencies:
- dependency-name: js-yaml
  dependency-version: 3.14.2
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump ajv from 6.12.6 to 6.14.0

Bumps [ajv](https://github.com/ajv-validator/ajv) from 6.12.6 to 6.14.0.
- [Release notes](https://github.com/ajv-validator/ajv/releases)
- [Commits](https://github.com/ajv-validator/ajv/compare/v6.12.6...v6.14.0)

---
updated-dependencies:
- dependency-name: ajv
  dependency-version: 6.14.0
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump axios in /wrongsecrets-balancer/ui

Bumps [axios](https://github.com/axios/axios) from 1.13.2 to 1.13.5.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.13.2...v1.13.5)

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.13.5
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump qs from 6.14.1 to 6.14.2 in /cleaner

Bumps [qs](https://github.com/ljharb/qs) from 6.14.1 to 6.14.2.
- [Changelog](https://github.com/ljharb/qs/blob/main/CHANGELOG.md)
- [Commits](https://github.com/ljharb/qs/compare/v6.14.1...v6.14.2)

---
updated-dependencies:
- dependency-name: qs
  dependency-version: 6.14.2
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

renovate[bot] committed:
chore(deps): update dependency axios to v1.13.5 [security]

renovate[bot] committed:
chore(deps): update pre-commit

renovate[bot] committed:
chore(deps): update dependency openid-client to v6.8.2

Copilot committed:
Initial plan

Copilot committed:
fix: update package-lock.json files for openid-client 6.8.2 and its dependencies

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump js-yaml from 3.14.1 to 3.14.2 in /cleaner

Bumps [js-yaml](https://github.com/nodeca/js-yaml) from 3.14.1 to 3.14.2.
- [Changelog](https://github.com/nodeca/js-yaml/blob/master/CHANGELOG.md)
- [Commits](https://github.com/nodeca/js-yaml/compare/3.14.1...3.14.2)

---
updated-dependencies:
- dependency-name: js-yaml
  dependency-version: 3.14.2
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump minimatch from 3.1.2 to 3.1.4

Bumps [minimatch](https://github.com/isaacs/minimatch) from 3.1.2 to 3.1.4.
- [Changelog](https://github.com/isaacs/minimatch/blob/main/changelog.md)
- [Commits](https://github.com/isaacs/minimatch/compare/v3.1.2...v3.1.4)

---
updated-dependencies:
- dependency-name: minimatch
  dependency-version: 3.1.4
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump minimatch in /wrongsecrets-balancer

Bumps [minimatch](https://github.com/isaacs/minimatch) from 3.1.2 to 10.2.3.
- [Changelog](https://github.com/isaacs/minimatch/blob/main/changelog.md)
- [Commits](https://github.com/isaacs/minimatch/compare/v3.1.2...v10.2.3)

---
updated-dependencies:
- dependency-name: minimatch
  dependency-version: 10.2.3
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump rollup in /wrongsecrets-balancer/ui

Bumps [rollup](https://github.com/rollup/rollup) from 2.79.2 to 2.80.0.
- [Release notes](https://github.com/rollup/rollup/releases)
- [Changelog](https://github.com/rollup/rollup/blob/v2.80.0/CHANGELOG.md)
- [Commits](https://github.com/rollup/rollup/compare/v2.79.2...v2.80.0)

---
updated-dependencies:
- dependency-name: rollup
  dependency-version: 2.80.0
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump minimatch in /wrongsecrets-balancer/ui

Bumps  and [minimatch](https://github.com/isaacs/minimatch). These dependencies needed to be updated together.

Updates `minimatch` from 3.1.2 to 3.1.5
- [Changelog](https://github.com/isaacs/minimatch/blob/main/changelog.md)
- [Commits](https://github.com/isaacs/minimatch/compare/v3.1.2...v3.1.5)

Updates `minimatch` from 9.0.5 to 9.0.9
- [Changelog](https://github.com/isaacs/minimatch/blob/main/changelog.md)
- [Commits](https://github.com/isaacs/minimatch/compare/v3.1.2...v3.1.5)

Updates `minimatch` from 5.1.6 to 5.1.9
- [Changelog](https://github.com/isaacs/minimatch/blob/main/changelog.md)
- [Commits](https://github.com/isaacs/minimatch/compare/v3.1.2...v3.1.5)

---
updated-dependencies:
- dependency-name: minimatch
  dependency-version: 3.1.5
  dependency-type: indirect
- dependency-name: minimatch
  dependency-version: 9.0.9
  dependency-type: indirect
- dependency-name: minimatch
  dependency-version: 5.1.9
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump fast-xml-parser in /wrongsecrets-balancer

Bumps [fast-xml-parser](https://github.com/NaturalIntelligence/fast-xml-parser) from 5.3.6 to 5.4.2.
- [Release notes](https://github.com/NaturalIntelligence/fast-xml-parser/releases)
- [Changelog](https://github.com/NaturalIntelligence/fast-xml-parser/blob/master/CHANGELOG.md)
- [Commits](https://github.com/NaturalIntelligence/fast-xml-parser/compare/v5.3.6...v5.4.2)

---
updated-dependencies:
- dependency-name: fast-xml-parser
  dependency-version: 5.4.2
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

renovate[bot] committed:
chore(deps): update terraform azurerm to v4.63.0

renovate[bot] committed:
chore(deps): update dependency axios to v1.13.6

renovate[bot] committed:
chore(deps): update dependency minimatch to v10.2.4

renovate[bot] committed:
chore(deps): update dependency google-auth-library to v10.6.1

renovate[bot] committed:
chore(deps): update dependency @eslint/eslintrc to v3.3.5

dependabot[bot] committed:
build(deps): bump qs from 6.14.1 to 6.15.0 in /wrongsecrets-balancer

Bumps [qs](https://github.com/ljharb/qs) from 6.14.1 to 6.15.0.
- [Changelog](https://github.com/ljharb/qs/blob/main/CHANGELOG.md)
- [Commits](https://github.com/ljharb/qs/compare/v6.14.1...v6.15.0)

---
updated-dependencies:
- dependency-name: qs
  dependency-version: 6.15.0
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

renovate[bot] committed:
chore(deps): update dependency react-router-dom to v7.13.1

dependabot[bot] committed:
build(deps): bump flatted from 3.3.3 to 3.4.1

Bumps [flatted](https://github.com/WebReflection/flatted) from 3.3.3 to 3.4.1.
- [Commits](https://github.com/WebReflection/flatted/compare/v3.3.3...v3.4.1)

---
updated-dependencies:
- dependency-name: flatted
  dependency-version: 3.4.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

renovate[bot] committed:
chore(deps): update dependency eslint to v9.39.4

renovate[bot] committed:
chore(deps): update dependency jest to v30.3.0

renovate[bot] committed:
chore(deps): update terraform azurerm to v4.64.0

renovate[bot] committed:
chore(deps): update toolmantim/release-drafter action to v6.4.0

renovate[bot] committed:
chore(deps): update actions/upload-pages-artifact action to v4

renovate[bot] committed:
chore(deps): update dependency @commitlint/config-conventional to v20

renovate[bot] committed:
chore(deps): update terraform-aws

dependabot[bot] committed:
build(deps): bump fast-xml-parser in /wrongsecrets-balancer

Bumps [fast-xml-parser](https://github.com/NaturalIntelligence/fast-xml-parser) from 5.4.2 to 5.5.6.
- [Release notes](https://github.com/NaturalIntelligence/fast-xml-parser/releases)
- [Changelog](https://github.com/NaturalIntelligence/fast-xml-parser/blob/master/CHANGELOG.md)
- [Commits](https://github.com/NaturalIntelligence/fast-xml-parser/compare/v5.4.2...v5.5.6)

---
updated-dependencies:
- dependency-name: fast-xml-parser
  dependency-version: 5.5.6
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump fast-xml-parser in /wrongsecrets-balancer

Bumps [fast-xml-parser](https://github.com/NaturalIntelligence/fast-xml-parser) from 5.5.6 to 5.5.7.
- [Release notes](https://github.com/NaturalIntelligence/fast-xml-parser/releases)
- [Changelog](https://github.com/NaturalIntelligence/fast-xml-parser/blob/master/CHANGELOG.md)
- [Commits](https://github.com/NaturalIntelligence/fast-xml-parser/compare/v5.5.6...v5.5.7)

---
updated-dependencies:
- dependency-name: fast-xml-parser
  dependency-version: 5.5.7
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump flatted from 3.3.3 to 3.4.2 in /cleaner

Bumps [flatted](https://github.com/WebReflection/flatted) from 3.3.3 to 3.4.2.
- [Commits](https://github.com/WebReflection/flatted/compare/v3.3.3...v3.4.2)

---
updated-dependencies:
- dependency-name: flatted
  dependency-version: 3.4.2
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Copilot committed:
Initial plan

dependabot[bot] committed:
build(deps-dev): bump jsonpath in /wrongsecrets-balancer/ui

Bumps [jsonpath](https://github.com/dchester/jsonpath) from 1.2.1 to 1.3.0.
- [Commits](https://github.com/dchester/jsonpath/commits)

---
updated-dependencies:
- dependency-name: jsonpath
  dependency-version: 1.3.0
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Copilot committed:
Update wrongsecrets and webtop containers to v1.13.1

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>

commjoen committed:
webtop up and running again

commjoen committed:
GEt challenge60 working as well

Copilot committed:
Initial plan

renovate[bot] committed:
chore(deps): update dependency google-auth-library to v10.6.2

renovate[bot] committed:
chore(deps): update dependency styled-components to v6.3.12

renovate[bot] committed:
chore(deps): update terraform aws to v6.37.0

renovate[bot] committed:
chore(deps): update terraform azurerm to v4.65.0

renovate[bot] committed:
chore(deps): update dependency globals to v17

renovate[bot] committed:
chore(deps): update dependency kubernetes-sigs/aws-alb-ingress-controller to v3

renovate[bot] committed:
chore(deps): update docker/build-push-action action to v7

renovate[bot] committed:
chore(deps): update docker/login-action action to v4

renovate[bot] committed:
chore(deps): update docker/metadata-action action to v6

renovate[bot] committed:
chore(deps): update docker/setup-buildx-action action to v4

renovate[bot] committed:
chore(deps): update docker/setup-qemu-action action to v4

renovate[bot] committed:
chore(deps): update github/codeql-action action to v4

renovate[bot] committed:
chore(deps): update hashicorp/setup-terraform action to v4

Copilot committed:
Fix CTF challenge creation by using configurable ctf_key from config instead of hardcoded value

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update pre-commit hook renovatebot/pre-commit-hooks to v43

renovate[bot] committed:
chore(deps): update terraform-gcp to v7

renovate[bot] committed:
chore(deps): update terraform-linters/setup-tflint action to v6

renovate[bot] committed:
chore(deps): update toolmantim/release-drafter action to v7

renovate[bot] committed:
fix(deps): update dependency googleapis to v171

renovate[bot] committed:
fix(deps): update react monorepo to v19

renovate[bot] committed:
fix(deps): update dependency eslint to v10

commjoen committed:
documentation update

dependabot[bot] committed:
build(deps): bump flatted from 3.4.1 to 3.4.2

Bumps [flatted](https://github.com/WebReflection/flatted) from 3.4.1 to 3.4.2.
- [Commits](https://github.com/WebReflection/flatted/compare/v3.4.1...v3.4.2)

---
updated-dependencies:
- dependency-name: flatted
  dependency-version: 3.4.2
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

renovate[bot] committed:
chore(deps): update dependency @eslint/js to v10

Copilot committed:
Initial plan

Copilot committed:
fix: add cause to re-thrown errors to satisfy ESLint v10 preserve-caught-error rule

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>
Agent-Logs-Url: https://github.com/OWASP/wrongsecrets-ctf-party/sessions/a8332975-fb76-42b8-8e62-eb786f00aebf

dependabot[bot] committed:
build(deps-dev): bump flatted in /wrongsecrets-balancer/ui

Bumps [flatted](https://github.com/WebReflection/flatted) from 3.3.3 to 3.4.2.
- [Commits](https://github.com/WebReflection/flatted/compare/v3.3.3...v3.4.2)

---
updated-dependencies:
- dependency-name: flatted
  dependency-version: 3.4.2
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

renovate[bot] committed:
chore(deps): update helm release kube-prometheus-stack to v82

Copilot committed:
Initial plan

Copilot committed:
fix: update README.md kube-prometheus-stack version to 82.13.6

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>
Agent-Logs-Url: https://github.com/OWASP/wrongsecrets-ctf-party/sessions/7e46fde8-da4e-4fdf-8309-875cdf3d5fc8

commjoen committed:
Release 1.13.2

dependabot[bot] committed:
build(deps-dev): bump flatted in /wrongsecrets-balancer

Bumps [flatted](https://github.com/WebReflection/flatted) from 3.3.3 to 3.4.2.
- [Commits](https://github.com/WebReflection/flatted/compare/v3.3.3...v3.4.2)

---
updated-dependencies:
- dependency-name: flatted
  dependency-version: 3.4.2
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Copilot committed:
Initial plan

commjoen commented:
any updates on this good sir ;) ?

commjoen committed:
Merge branch 'main' into fix/ci-e2e-script

Copilot committed:
Add challenge62 Google service account support: secret, configmap, and documentation

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>
Agent-Logs-Url: https://github.com/OWASP/wrongsecrets-ctf-party/sessions/e7c14ffb-7ab8-4460-bc11-6f2507fb2877

Copilot committed:
Address code review: rename variable and simplify string handling

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>
Agent-Logs-Url: https://github.com/OWASP/wrongsecrets-ctf-party/sessions/e7c14ffb-7ab8-4460-bc11-6f2507fb2877

Copilot committed:
Update helm README via helm-docs to include challenge62 values

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>
Agent-Logs-Url: https://github.com/OWASP/wrongsecrets-ctf-party/sessions/d5b41117-190a-4147-9cfe-0b04ebc30139

Copilot committed:
Fix README footer: regenerate helm docs with v1.11.0 to match CI workflow

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>
Agent-Logs-Url: https://github.com/OWASP/wrongsecrets-ctf-party/sessions/5fab917d-1f17-4555-95c7-344d3fac67ee

dependabot[bot] committed:
build(deps): bump picomatch in /wrongsecrets-balancer

Bumps  and [picomatch](https://github.com/micromatch/picomatch). These dependencies needed to be updated together.

Updates `picomatch` from 2.3.1 to 2.3.2
- [Release notes](https://github.com/micromatch/picomatch/releases)
- [Changelog](https://github.com/micromatch/picomatch/blob/master/CHANGELOG.md)
- [Commits](https://github.com/micromatch/picomatch/compare/2.3.1...2.3.2)

Updates `picomatch` from 4.0.3 to 4.0.4
- [Release notes](https://github.com/micromatch/picomatch/releases)
- [Changelog](https://github.com/micromatch/picomatch/blob/master/CHANGELOG.md)
- [Commits](https://github.com/micromatch/picomatch/compare/2.3.1...2.3.2)

---
updated-dependencies:
- dependency-name: picomatch
  dependency-version: 2.3.2
  dependency-type: indirect
- dependency-name: picomatch
  dependency-version: 4.0.4
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump picomatch from 2.3.1 to 2.3.2 in /cleaner

Bumps [picomatch](https://github.com/micromatch/picomatch) from 2.3.1 to 2.3.2.
- [Release notes](https://github.com/micromatch/picomatch/releases)
- [Changelog](https://github.com/micromatch/picomatch/blob/master/CHANGELOG.md)
- [Commits](https://github.com/micromatch/picomatch/compare/2.3.1...2.3.2)

---
updated-dependencies:
- dependency-name: picomatch
  dependency-version: 2.3.2
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump node-forge in /wrongsecrets-balancer/ui

Bumps [node-forge](https://github.com/digitalbazaar/forge) from 1.3.2 to 1.4.0.
- [Changelog](https://github.com/digitalbazaar/forge/blob/main/CHANGELOG.md)
- [Commits](https://github.com/digitalbazaar/forge/compare/v1.3.2...v1.4.0)

---
updated-dependencies:
- dependency-name: node-forge
  dependency-version: 1.4.0
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump picomatch in /wrongsecrets-balancer/ui

Bumps [picomatch](https://github.com/micromatch/picomatch) from 2.3.1 to 2.3.2.
- [Release notes](https://github.com/micromatch/picomatch/releases)
- [Changelog](https://github.com/micromatch/picomatch/blob/master/CHANGELOG.md)
- [Commits](https://github.com/micromatch/picomatch/compare/2.3.1...2.3.2)

---
updated-dependencies:
- dependency-name: picomatch
  dependency-version: 2.3.2
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump path-to-regexp in /wrongsecrets-balancer

Bumps [path-to-regexp](https://github.com/pillarjs/path-to-regexp) from 8.3.0 to 8.4.0.
- [Release notes](https://github.com/pillarjs/path-to-regexp/releases)
- [Changelog](https://github.com/pillarjs/path-to-regexp/blob/master/History.md)
- [Commits](https://github.com/pillarjs/path-to-regexp/compare/v8.3.0...v8.4.0)

---
updated-dependencies:
- dependency-name: path-to-regexp
  dependency-version: 8.4.0
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

renovate[bot] committed:
chore(deps): update dependency react-router-dom to v7.13.2

renovate[bot] committed:
chore(deps): update dependency axios to v1.14.0

renovate[bot] committed:
chore(deps): update dependency eslint to v10.1.0

renovate[bot] committed:
chore(deps): update pre-commit hook renovatebot/pre-commit-hooks to v43.102.2

renovate[bot] committed:
chore(deps): update pre-commit hook renovatebot/pre-commit-hooks to v43.102.3

renovate[bot] committed:
chore(deps): update terraform azurerm to v4.66.0

renovate[bot] committed:
chore(deps): update pre-commit hook renovatebot/pre-commit-hooks to v43.102.8

renovate[bot] committed:
chore(deps): update terraform-gcp to ~> 7.26.0

renovate[bot] committed:
chore(deps): update actions/configure-pages action to v6

renovate[bot] committed:
chore(deps): update dependency minimatch to v10.2.5

renovate[bot] committed:
chore(deps): update dependency lodash to v4.18.1

renovate[bot] committed:
chore(deps): update terraform aws to v6.39.0

renovate[bot] committed:
chore(deps): update actions/deploy-pages action to v5

dependabot[bot] committed:
build(deps): bump lodash in /wrongsecrets-balancer/ui

Bumps [lodash](https://github.com/lodash/lodash) from 4.17.23 to 4.18.1.
- [Release notes](https://github.com/lodash/lodash/releases)
- [Commits](https://github.com/lodash/lodash/compare/4.17.23...4.18.1)

---
updated-dependencies:
- dependency-name: lodash
  dependency-version: 4.18.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

renovate[bot] committed:
chore(deps): update dependency axios to v1.15.0 [security]

dependabot[bot] committed:
build(deps): bump axios in /wrongsecrets-balancer/ui

Bumps [axios](https://github.com/axios/axios) from 1.14.0 to 1.15.0.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.14.0...v1.15.0)

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.15.0
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump follow-redirects in /wrongsecrets-balancer

Bumps [follow-redirects](https://github.com/follow-redirects/follow-redirects) from 1.15.11 to 1.16.0.
- [Release notes](https://github.com/follow-redirects/follow-redirects/releases)
- [Commits](https://github.com/follow-redirects/follow-redirects/compare/v1.15.11...v1.16.0)

---
updated-dependencies:
- dependency-name: follow-redirects
  dependency-version: 1.16.0
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump protobufjs in /wrongsecrets-balancer

Bumps [protobufjs](https://github.com/protobufjs/protobuf.js) from 7.5.4 to 7.5.5.
- [Release notes](https://github.com/protobufjs/protobuf.js/releases)
- [Changelog](https://github.com/protobufjs/protobuf.js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/protobufjs/protobuf.js/compare/protobufjs-v7.5.4...protobufjs-v7.5.5)

---
updated-dependencies:
- dependency-name: protobufjs
  dependency-version: 7.5.5
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

renovate[bot] committed:
chore(deps): update dependency eslint to v10.2.1

renovate[bot] committed:
chore(deps): update dependency kubernetes-sigs/aws-alb-ingress-controller to v3.2.2

dependabot[bot] committed:
build(deps): bump follow-redirects in /wrongsecrets-balancer/ui

Bumps [follow-redirects](https://github.com/follow-redirects/follow-redirects) from 1.15.11 to 1.16.0.
- [Release notes](https://github.com/follow-redirects/follow-redirects/releases)
- [Commits](https://github.com/follow-redirects/follow-redirects/compare/v1.15.11...v1.16.0)

---
updated-dependencies:
- dependency-name: follow-redirects
  dependency-version: 1.16.0
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

renovate[bot] committed:
chore(deps): update dependency prettier to v3.8.3

renovate[bot] committed:
chore(deps): update react monorepo to v19.2.5

renovate[bot] committed:
chore(deps): update dependency globals to v17.5.0

dependabot[bot] committed:
build(deps): bump fast-xml-parser in /wrongsecrets-balancer

Bumps [fast-xml-parser](https://github.com/NaturalIntelligence/fast-xml-parser) from 5.5.7 to 5.7.1.
- [Release notes](https://github.com/NaturalIntelligence/fast-xml-parser/releases)
- [Changelog](https://github.com/NaturalIntelligence/fast-xml-parser/blob/master/CHANGELOG.md)
- [Commits](https://github.com/NaturalIntelligence/fast-xml-parser/compare/v5.5.7...v5.7.1)

---
updated-dependencies:
- dependency-name: fast-xml-parser
  dependency-version: 5.7.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

renovate[bot] committed:
chore(deps): update helm release kube-prometheus-stack to v82.18.0

Copilot committed:
Add reference to CHALLENGE62_GOOGLE_DRIVE_SETUP.md in challenge62 documentation

Agent-Logs-Url: https://github.com/OWASP/wrongsecrets-ctf-party/sessions/f2f56e28-d430-4d39-a97b-78daa1c1407e

Co-authored-by: commjoen <1457214+commjoen@users.noreply.github.com>

commjoen committed:
Merge branch 'main' into fix/ci-e2e-script

seershan committed:
fix(ci): run deployment script synchronously

seershan committed:
fix(ci): improve deployment readiness checks

seershan committed:
fix(ci): improve minikube deployment workflow

seershan committed:
fix(ci): avoid deployment race condition

seershan committed:
fix(ci): add Cypress dependency and Node 24 setup

seershan committed:
fix(ci): regenerate lockfile
